lib/rubocop/cop/generator.rb

```diff
@@ -7,7 +7,7 @@ module Cop
     # This generator will take a cop name and generate a source file
     # and spec file when given a valid qualified cop name.
     class Generator
-      SOURCE_TEMPLATE = <<-RUBY.strip_indent
+      SOURCE_TEMPLATE = <<-RUBY.strip_indent.freeze
```

Comment:
Heredocs are affected by `# frozen_string_literal: true`, so this looks like a false positive. 🙂

Reply:
`strip_indent` mutates the string and returns a non-frozen string so calling `freeze` is necessary.

Reply:
Had to have a look at Powerpack. Seems a new string is returned, but since it's not a literal, it's also not frozen. You're correct. 🙂